dpypelines/pipeline/dataset_ingress_v1.py

```diff
@@ -95,57 +103,69 @@ def dataset_ingress_v1(files_dir: str, pipeline_config: dict):
                 )
                 raise ValueError(msg)
         except Exception as err:
+            logger.error(f"Error while looking for supplementary distribution {supplementary_distribution}", 
+            err, 
+            data={"supplementary_distribution": supplementary_distribution}
+            )
+
             de_notifier.failure()
-            raise Exception(
-                message.unexpected_error(
-                    f"Error while looking for supplementary distribution {supplementary_distribution}",
-                    err,
-                )
-            ) from err
+            raise err
 
     # Get the positional arguments (the inputs) from the pipeline_config
     # dict and run the specified sanity checker for it
     args = []
     for match, sanity_checker in pipeline_config["transform_inputs"].items():
         try:
             input_file_path: Path = local_store.save_lone_file_matching(match)
+            logger.info(f"Successfully saved file that matches pattern to path {input_file_path}", data={"input_file_path": input_file_path})
```

Comment:
good idea 👍 , but you dont need the variable in the message as you're capturing it with the data= kwarg.

I would probably also log out the match variable with the data kwarg too, so we can see at a glance which pattern resulted in which file being saved.